<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>naver</class>
 <widget class="QMainWindow" name="naver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>네이버 검색</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>541</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Search</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>검색어</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSearchWord">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>261</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSearch">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tblResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>451</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="stsResult"/>
 </widget>
 <resources/>
 <connections/>
</ui>
